src/Command/Entity/ContentEntity.php

```diff
@@ -42,6 +42,7 @@ protected function generate(array &$vars, AssetCollection $assets): void {
     $vars['entity_base_path'] = $ir->ask('Entity base path', '/{entity_type_id_short|u2h}');
     $vars['fieldable'] = $ir->confirm('Make the entity type fieldable?');
     $vars['revisionable'] = $ir->confirm('Make the entity type revisionable?', FALSE);
+    $vars['revision_ui'] = $vars['revisionable'] && $ir->confirm('Show revision UI?');
```

Comment:
Can we remove this confirmation? Having revisions without UI doesn't make much sense.